Index: plugins/mpsdevkit/languages/pluginSolutions/lang.migration/models/jetbrains/mps/lang/migration/pluginSolution/plugin.mps
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- plugins/mpsdevkit/languages/pluginSolutions/lang.migration/models/jetbrains/mps/lang/migration/pluginSolution/plugin.mps	(date 1443142750000)
+++ plugins/mpsdevkit/languages/pluginSolutions/lang.migration/models/jetbrains/mps/lang/migration/pluginSolution/plugin.mps	(revision )
@@ -125,6 +125,7 @@
         <child id="1081773367580" name="leftExpression" index="3uHU7B" />
       </concept>
       <concept id="1073239437375" name="jetbrains.mps.baseLanguage.structure.NotEqualsExpression" flags="nn" index="3y3z36" />
+      <concept id="1208623485264" name="jetbrains.mps.baseLanguage.structure.AbstractOperation" flags="nn" index="1B$H19" />
       <concept id="6329021646629104957" name="jetbrains.mps.baseLanguage.structure.TextCommentPart" flags="nn" index="3SKdUq">
         <property id="6329021646629104958" name="text" index="3SKdUp" />
       </concept>
@@ -203,6 +204,7 @@
       <concept id="7738379549910147341" name="jetbrains.mps.lang.smodel.query.structure.InstancesExpression" flags="ng" index="qVDSY">
         <child id="7738379549910147342" name="conceptArg" index="qVDSX" />
       </concept>
+      <concept id="6864030874028745314" name="jetbrains.mps.lang.smodel.query.structure.ModulesExpression" flags="ng" index="EzsRk" />
       <concept id="4234138103881610891" name="jetbrains.mps.lang.smodel.query.structure.WithStatement" flags="ng" index="L3pyB">
         <child id="4234138103881610935" name="scope" index="L3pyr" />
         <child id="4234138103881610892" name="stmts" index="L3pyw" />
@@ -219,6 +221,7 @@
       <concept id="540871147943773365" name="jetbrains.mps.baseLanguage.collections.structure.SingleArgumentSequenceOperation" flags="nn" index="25WWJ4">
         <child id="540871147943773366" name="argument" index="25WWJ7" />
       </concept>
+      <concept id="1204980550705" name="jetbrains.mps.baseLanguage.collections.structure.VisitAllOperation" flags="nn" index="2es0OD" />
       <concept id="1226511727824" name="jetbrains.mps.baseLanguage.collections.structure.SetType" flags="in" index="2hMVRd">
         <child id="1226511765987" name="elementType" index="2hN53Y" />
       </concept>
@@ -1028,7 +1031,49 @@
     <property role="TrG5h" value="ExecuteRerunnableMigrations" />
     <property role="2uzpH1" value="Execute Re-Runnable Migrations" />
     <node concept="tnohg" id="5$zfhXzhLy2" role="tncku">
-      <node concept="3clFbS" id="5$zfhXzhLy3" role="2VODD2" />
+      <node concept="3clFbS" id="5$zfhXzhLy3" role="2VODD2">
+        <node concept="L3pyB" id="5$zfhXzjQ_R" role="3cqZAp">
+          <node concept="2OqwBi" id="5$zfhXzjQCz" role="L3pyr">
+            <node concept="2WthIp" id="5$zfhXzjQCA" role="2Oq$k0" />
+            <node concept="1DTwFV" id="5$zfhXzjQCC" role="2OqNvi">
+              <ref role="2WH_rO" node="5$zfhXzjQau" resolve="mpsProject" />
+            </node>
+          </node>
+          <node concept="3clFbS" id="5$zfhXzjQ_T" role="L3pyw">
+            <node concept="3clFbH" id="5$zfhXzjXaJ" role="3cqZAp" />
+            <node concept="3clFbF" id="5$zfhXzjQK1" role="3cqZAp">
+              <node concept="2OqwBi" id="5$zfhXzjQTs" role="3clFbG">
+                <node concept="EzsRk" id="5$zfhXzjQJZ" role="2Oq$k0">
+                  <node concept="1dO9Bo" id="5$zfhXzjQK0" role="1dOa5D" />
+                </node>
+                <node concept="2es0OD" id="5$zfhXzjRqo" role="2OqNvi">
+                  <node concept="1bVj0M" id="5$zfhXzjRqq" role="23t8la">
+                    <node concept="3clFbS" id="5$zfhXzjRqr" role="1bW5cS">
+                      <node concept="3clFbF" id="5$zfhXzjRuV" role="3cqZAp">
+                        <node concept="2OqwBi" id="5$zfhXzjRwY" role="3clFbG">
+                          <node concept="37vLTw" id="5$zfhXzjRuU" role="2Oq$k0">
+                            <ref role="3cqZAo" node="5$zfhXzjRqs" resolve="it" />
+                          </node>
+                          <node concept="1B$H19" id="5$zfhXzjRx3" role="2OqNvi" />
+                        </node>
+                      </node>
+                    </node>
+                    <node concept="Rh6nW" id="5$zfhXzjRqs" role="1bW2Oz">
+                      <property role="TrG5h" value="it" />
+                      <node concept="2jxLKc" id="5$zfhXzjRqt" role="1tU5fm" />
+                    </node>
+                  </node>
+                </node>
+              </node>
+            </node>
+          </node>
+        </node>
+      </node>
+    </node>
+    <node concept="1DS2jV" id="5$zfhXzjQau" role="1NuT2Z">
+      <property role="TrG5h" value="mpsProject" />
+      <ref role="1DUlNI" to="qq03:~MPSCommonDataKeys.MPS_PROJECT" resolve="MPS_PROJECT" />
+      <node concept="1oajcY" id="5$zfhXzjQav" role="1oa70y" />
     </node>
   </node>
 </model>
